.expeditor/config.yml

```diff
@@ -0,0 +1,16 @@
+slack:
+  notify_channel: omnibus-rfr
+
+github:
+  maintainer_group: chef/omnibus-maintainers
+  version_tag_format: "v{{version}}"
+
+merge_actions:
+  - built_in:bump_version:
+      ignore_labels:
+        - "Version: Skip Bump"
```

Comment:
Idiomatically we usually add a `Expeditor: Skip All` under each merge action. I wonder if we should consider adding this as built-in behavior @tduffield ?

Reply:
@schisamo I go back and forth on that. My initial design intention was to be very explicit in this YAML about what labels do what.